xla_native_functions.yaml

```diff
@@ -321,6 +321,7 @@ supported:
   - triangular_solve
   - unbind.int
   - uniform_
+  - _unique2
```

Comment:
I wonder how `_unique2` differs from `unique`

Reply:
The `_unique` [function](https://github.com/pytorch/pytorch/blob/e22d791287d1c00cd11b9229ad15d030224ab4b1/aten/src/ATen/native/native_functions.yaml#L5842) does not support `return_counts`. It looks like it's not accessible to normal user because `torch.unique` is dispatched to either `unique_dim` or `_unique2` [here](https://github.com/pytorch/pytorch/blob/1e2d82b8e436fc5c72071c3b78bc9d13f4a02268/torch/functional.py#L722-L724).